### ISSUE TITLE
Sorry, would you like to ask if this project will continue to be updated?

### ISSUE THREAD

**Issue by scx1125** [chinware/AtomUI#18]
Sorry, would you like to ask if this project will continue to be updated?

**Comment by chinware** [comment]
Yes, this project will continue to be maintained. In the past two months, I have focused on developing the official website. The AtomUI manual will be available on the official website.

**Comment by chinware** [comment]
@scx1125 
Yes, this project will continue to be maintained. In the past two months, I have focused on developing the official website. The AtomUI manual will be available on the official website.
